keyboards/lets_split/keymaps/aerialviews007/keymap.c

```diff
@@ -0,0 +1,159 @@
+#include QMK_KEYBOARD_H
+
+// Each layer gets a name for readability, which is then used in the keymap matrix below.
+// The underscores don't mean anything - you can have a layer called STUFF or any other name.
+// Layer names don't all need to be of the same length, obviously, and you can also skip them
+// entirely and just use numbers.
+#define _QWERTY 0
+#define _COLEMAK 1
+#define _DVORAK 2
+#define _LOWER 3
+#define _RAISE 4
+#define _ADJUST 16
```

Comment:
We recommend using an enum for the layers rather than defines. For the most part, they work the same, but enums auto-increment, making it easier to manage, long term. 

Also, having adjust at layer 16 means that it adds a bunch of blank layers, and increases the firmware size for no real gain. 

```suggestion
enum custom_layers {
    _QWERTY,
    _COLEMAK,
    _DVORAK,
    _LOWER,
    _RAISE,
    _ADJUST,
};
```